### DISCUSSION TITLE
https://thanhle.blog/blog/what-is-the-difference-between-hobby-and-passion

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#17] [Ideas]
# https://thanhle.blog/blog/what-is-the-difference-between-hobby-and-passion

https://thanhle.blog/blog/what-is-the-difference-between-hobby-and-passion

**Comment by darkpiv** [comment]
a nghĩ là chia ra thì ko quan trọng lắm, kiểu cứ flat nó ra giống miếng bánh pizza chia ra :D thì cái nào nhìn cũng same same, nhưng mà đã chia ra và picked rồi thì phải focus vào đó làm cho tốt, đặc biệt ko lơ là mấy miếng type "pay my bills" và "pay my future bill" :D Khi nào tới level đảm bảo bill luôn có thể auto pay thì tới đó tính tiếp :D. Chắc là mở Lớp bộ môn public để mn cũng học :D 
Hobbies thì cũng như e, quá trời thứ :3, riêng passion thì nhìn thấy code chạy, user xài, add features, optimize perf cho user tiếp tục xài là thấy vui nhất, BE, FE hay DevOps gì cũng được. :D

**Reply by thanhlmm** [reply]
Nice Idea! Nhưng sẽ đến lúc nhận ra có quá nhiều miếng bách để pick? Vậy nên chọn miếng nào :3 Việc hiểu được bản thân đón nhận challenges như thế nào là cách e thấy khá ổn để chọn pick miếng nào
